### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by dereklieu**
Takes care of a lot of smaller ticket items.

Over to @kamicut to review and merge.

### CODE REVIEW

File: app/assets/scripts/views/project.js [hunk 1]
```diff
@@ -190,26 +190,32 @@ var Project = React.createClass({
...
+                {data.responsible_ministry && data.responsible_ministry.toLowerCase() !== 'select a ministry' && (
+                  <div className='overview-item'>
+                    <h2 className='overview-item__title heading-alt'>Responsible Ministry</h2>
+                    <ul className='link-list'>
+                      <li><a href='' className='link--primary'><span>{data.responsible_ministry}</span></a></li>
+                    </ul>
+                  </div>
+                )}
+
+                {data.responsible_party && data.responsible_party.toLowerCase() !== 'select a party' && (
```
Comment (kamicut) [inline app/assets/scripts/views/project.js]: I don't think there's a responsible party in the data schema, what is this referring to?

Comment (dereklieu) [inline app/assets/scripts/views/project.js]: I'm pretty sure this is different in the final version.

File: app/assets/scripts/views/project.js [hunk 2]
```diff
@@ -189,31 +190,50 @@ var Project = React.createClass({
...
+                    <h2 className='overview-item__title heading-alt'>Local Manager</h2>
+                    <ul className='link-list'>
+                      <li><a href='' className='link--primary'><span>{data.local_manager}</span></a></li>
+                    </ul>
+                  </div>
+                )}
 
-                <div className='overview-item'>
-                  <h2 className='overview-item__title heading-alt'>Responsible Party</h2>
+                <div className='overview-item--alt'>
```
Comment (kamicut) [inline app/assets/scripts/views/project.js]: Should this header be hidden if there are no KMI components?

Comment (dereklieu) [inline app/assets/scripts/views/project.js]: Good question, not actually sure if we want to hide the entire thing or show a 'NA' or '--' or other sign to indicate there's no data. Leaving this up to @felskia.

**Pull request merged**